doc/06-Postprocessing.md

```diff
@@ -141,10 +149,35 @@ To export your variables to a file, `vfc_probes` relies on the
 your code through a Verificarlo CI test run, so you usually won't have to care
 about it,  but if you were to manually execute a program that calls the
 `vfc_dump_probes` function without this variable, you would be notified by a
-runtime warning explaining that your probes canno tbe exported.
+runtime warning explaining that your probes cannot be exported.
+
+Finally, probes can be used with an optional "check". Checks are accuracy
+targets that we want to reach on test variables. If a probe is created with a
+check, the tool will estimate its error and compare it to the specified
+accuracy target. However, the definition of this error can vary depending on the
+backend and the check types. Backends are separated in two categories, non-
+deterministic (such as the MCA backend) and deterministic (such as the VPREC backend),
+and checks precision can be absolute (by default) or relative. This results in four
+different conditions to validate the probe, which are summed up in the following
+table :
+
+| |Absolute|Relative|
+--- | --- | ---
+|Non-deterministic|\|Standard deviation\| < Target | \|Standard deviation\| / \|Empirical average\| < Target
+|Deterministic|\|IEEE value\| -  \|Backend value\| < Target|(\|IEEE value\| -  \|Backend value\|) / \|IEEE value\| < Target
+
+By default, the standard deviation is used to estimate the error, and must be
+inderior to the accuracy threshold for the probe to pass (or inferior to the
+std. dev. / avg. quotient in the case of a relative check).
+But in the case of a deterministic backend, it is impossible to compute the standard
```

Comment:
"it is impossible to compute the standard deviation" -> "the standard deviation is undefined"